dsgrid/config/dimensions.py

```diff
@@ -450,9 +467,62 @@ def is_time_zone_required_in_geography(self):
         """Returns True if the geography dimension records must contain a time_zone column."""
 
 
+class AlignedTime(DSGBaseModel):
+    """Data has absolute timestamps that are aligned with the same start and end
+    for each geography."""
+
+    format_type: Literal[DatetimeFormat.ALIGNED] = DatetimeFormat.ALIGNED
+    timezone: Annotated[
+        TimeZone,
+        Field(
+            title="timezone",
+            description="Time zone of data",
+            json_schema_extra={
+                "options": TimeZone.format_descriptions_for_docs(),
+            },
+        ),
+    ]
+
+
+class LocalTimeAsStrings(DSGBaseModel):
+    """Data has absolute timestamps formatted as strings with offsets from UTC.
+    They are aligned for each geography when adjusted for time zone but staggered
+    in an absolute time scale."""
+
+    format_type: Literal[DatetimeFormat.LOCAL_AS_STRINGS] = DatetimeFormat.LOCAL_AS_STRINGS
+
+    data_str_format: Annotated[
+        Optional[str],
```

Comment:
Has a default value, doesn't that make it optional?

Reply:
This is a frequently confusing topic. `Optional` in this case it is equivalent to `Union[str, None]`. That means that `None` is allowed and all of the code processing it has to account for that fact. Providing a default value that is a `str` means it will always be a `str`.